app/models/ebean/ChromeScreenshotReportSource.java

```diff
@@ -43,6 +44,12 @@
     @Column(name = "ignore_certificate_errors")
     private boolean ignoreCertificateErrors;
 
+    @Column(name = "timeout")
+    private Duration timeout;
```

Comment:
I believe there is a problem with `Duration` serialization through EBean where it truncates the seconds. 

https://github.com/ArpNetworking/metrics-portal/pull/179/files#r279255056